module/mailnews/mailnews.c

```diff
@@ -291,7 +291,7 @@ Bool HandleLookNewsgroup(char *ptr, long len)
    Extract(&ptr, &newsgroup, SIZE_NEWSGROUP_ID);
    Extract(&ptr, &permission, 1);
 
-   ExtractObject(&ptr, &obj);
+   ExtractObject(&ptr, &obj, true);
```

Comment:
No need to change these once the default value is in the header.

Reply:
Reverted to remove the `true` parameter.